docs/DatabaseChecks.md

```diff
@@ -36,4 +36,16 @@ If the config value is set to false the check will be skipped.
 
 ```json
 "CheckForOversizedIndexes": true
-```
\ No newline at end of file
+```
+
+## Fixed-Size Filegroups  
+Reports filegroups that are not set to auto-grow. You can whitelist fixed size filegroups by adding the name to the config array
+
+```json
+"ZeroAutoGrowthWhitelistFGs":[
+        "AdventureWorks2016CTP3_mod"
+        ,"templog"]
+```
+
+## Auto-growth & at-risk Filegroups
+Filegroups that are permitted to auto-grow should have enough space to do so. `Get-AutoGrowthRisks` reports filegroups that may run out of space and fail to complete the next autogrowth. No whitelist configuration is provided for this check. Set growth to `0` if you wish to disallow further growth actions.   
```

Comment:
What is the config value to enable/disable the check?

Reply:
As above, allowing for a whitelist in the pester felt like premature optimization when writing. I couldn't think of a case where you'd *want* to be in that position - and testing for the behavior of the whitelist config felt like a waste of time. Thoughts?